src/test/lambda/extensionManifest.test.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as packageJson from '../../../package.json'
+import * as packageNlsJson from '../../../package.nls.json'
+import * as lodash from 'lodash'
+
+describe('Test Copy Lambda Func URL in extension manifest (package.json)', async () => {
```

Comment:
- avoid verbose names.
- tests don't need to mention "test" in the name.

```suggestion
describe('package.json', async () => {
```

Here the describe() block is just "package.json", because the it() blocks below add specific context.